--- a/cmd2.py
+++ b/cmd2.py
@@ -47,6 +47,13 @@
 if sys.version_info[0] == 2:
     pyparsing.ParserElement.enablePackrat()
 
+try:
+    is_mswindows = subprocess.mswindows
+except AttributeError:
+    # Python 3.5+
+    is_mswindows = subprocess._mswindows
+
+
 """
 Packrat is causing Python3 errors that I don't understand.
 
@@ -184,7 +191,7 @@
 to be installed on operating system.
 %s"""
 
-if subprocess.mswindows:
+if is_mswindows:
     try:
         import win32clipboard
         def get_paste_buffer():
